examples/common/synchronous-client.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
```

Comment:
There is alreadty a file by name `synchronous_client.py` , Remove this from the commit.